lbry/lbry/extras/daemon/storage.py

```diff
@@ -273,6 +273,14 @@ class SQLiteStorage(SQLiteMixin):
                 timestamp integer,
                 primary key (sd_hash, reflector_address)
             );
+
+            create table if not exists peer (
```

Comment:
this can be cleaner with `node_id` as the primary key and a `unique (address, udp_port)` constraint

Reply:
Agreed.